complete/src/main/java/dao/RetrofitClientInstance.java

```diff
@@ -0,0 +1,19 @@
+
+import retrofit2.Retrofit
+
+public class RetrofitClientInstance {
+
+    private static Retrofit retrofit;
+
+    private static String Base_URL = "";
+
+    public static Retrofit getRetrofitInstance() {
+        if (retrofit == null) {
+            retrofit = new retrofit2.Retrofit.Builder()
+                    .baseUrl(Base_URL)
+                    .addLocation(GsonLocation.create())
+                    .build();
+        }
+        return retrofit;
+    }
```

Comment:
Added a retrofit instance for authenticating users.

Source - https://www.tutorialspoint.com/java/java_networking.htm

Reply:
Ditto above.  Is it used anywhere? The BASE_URL is empty, which makes me think it might not be.

A good idea, but complete it before merging to main/master.